Templates/integration_template.ps1

```diff
@@ -0,0 +1,83 @@
+<#
+.Synopsis
+   Template for creating DSC Resource Integration Tests
+.DESCRIPTION
+   To Use:
+     1. Copy to \Tests\Integration\ folder and rename MSFT_x<ResourceName>.Integration.tests.ps1
+     2. Customize TODO sections.
+     3. Create test DSC Configurtion file MSFT_x<ResourceName>.config.ps1 from integration_config_template.ps1 file.
+
+.NOTES
+   Code in HEADER, FOOTER and DEFAULT TEST regions are standard and may be moved into
+   DSCResource.Tools in Future and therefore should not be altered if possible.
+#>
+
+# TODO: Customize these paramters...
+$DSCModuleName      = 'x<ModuleName>' # Example xNetworking
+$DSCResourceName    = 'MSFT_x<ResourceName>' # Example MSFT_xFirewall
+# /TODO
+
+#region HEADER
+if ( (-not (Test-Path -Path '.\DSCResource.Tests\')) -or `
```

Comment:
I think instead of doing this we could do a Get-Module DSCResource.Tests and import if it exists, and if it doesn't then throw this. This way we are not demanding that we have a clone of the files in the repo but can still excercise the function.